README.md

```diff
@@ -170,7 +167,54 @@ curl --user <username>:<password> http://localhost:3000//api/v1/items -X POST -F
   ```
  
 
+### INDEX Reviews
 
+  Returns json data about all reviews associated with a particular item.
+
+* **URL**
+
+  /api/v1/items/:id/reviews
+
+* **Method:**
+
+  `GET`
+  
+*  **URL Params**
+
+   **Required:**
+ 
+   `id=[integer]`
+
+* **Data Params**
+
+  None
+* **Sample cURL Request:**
+
+```bash
+curl https://localhost:3000.com/api/v1/items/3/reviews | jq
+```
+
+* **Success Response:**
+
+  * **Code:** 200
+ 
+    **Content:**
+  ```javascript
+  [
+    {
+      "text": "this made my dog sick"
```

Comment:
Based on the code I'm seeing, this seems incorrect. Wouldn't we be returning all the review attributes in the response?